### DISCUSSION TITLE
how to bundle lodash with endo?

### DISCUSSION THREAD

**Discussion by dckc** [endojs/endo#2438] [Q&A]
We're hitting
```
TypeError#2: Cannot read properties of undefined (reading 'Array')
```

context: https://github.com/Agoric/agoric-sdk/discussions/9986#discussioncomment-10478034

_TODO: copy relevant parts to make this a more complete question_

_If I had steps to reproduce without using agoric-sdk, I would have made an issue rather than a discussion. But I don't, yet._

cc @erights @kriskowal @naugtur

**Comment by dckc** [comment]
@erights writes:
> ...
> But the closest I could find within the lodash sources is already a corrected version that would work instead.
> https://github.com/lodash/lodash/blob/main/src/.internal/root.ts
> ...
> But lodash already has source code for the correct version, that also
> checks globalThis. Why isn’t it using that one?

Because it's not released?

`.internal/root.ts` says `0b28b7f` Sep 13, 2023
but lodash 4.17.21 was "Published 4 years ago" Feb 20, 2021. Likewise lodash-es 4.17.21.

**Reply by erights** [reply]
> Because it's not released?

Thanks.
